docs/developer-guide/etendo-classic/bundles/platform/etendo-webhooks.md

```diff
@@ -88,35 +95,57 @@ The access tab allows you to create access that will be used in the URL call.
 ![Webhook Token](../../../../assets/developer-guide/etendo-classic/bundles/platform/etendo-webhooks/WebhookToken.png)
```

Comment:
![Screenshot 2024-05-02 at 2 40 52 PM](https://github.com/etendosoftware/docs/assets/106354215/26e96a85-be1e-4a3a-a889-babbb0fbd486)
This screenshot is more descriptive